packages/botonic-react/src/components/button.jsx

```diff
@@ -63,3 +63,16 @@ export const Button = props => {
   if (isBrowser()) return renderBrowser()
   else if (isNode()) return renderNode()
 }
+
+Button.serialize = buttonProps => {
+  let payload = buttonProps.payload
+  if (buttonProps.path) payload = `__PATH_PAYLOAD__${buttonProps.path}`
+  return {
+    button: {
+      payload,
+      url: buttonProps.url,
+      webview: buttonProps.webview,
```

Comment:
Webview is a react component which is not serializable. Not sure how to deal with this. Maybe it would be reasonable to leave it null for now, seems to me that refreshing the screen and buttons not keeping the link to webview is not that big deal. We can look into it in future versions.

Reply:
Right now we can leave it null. Maybe in the future we can try to store the path where we are importing this webview ( need to investigate how to get it from the react component, if posible )